src/Tribe/Dependency.php

```diff
@@ -44,13 +60,14 @@ public static function instance() {
 		/**
 		 * Adds a plugin to the active list
 		 *
-		 * @since tbd
+		 * @since TBD
 		 *
-		 * @param string $main_class Main/base class for this plugin
-		 * @param string $version    Version number of plugin
-		 * @param string $path       Path to the main plugin/bootstrap file
+		 * @param string $main_class    Main/base class for this plugin
+		 * @param string $path          Path to the main plugin/bootstrap file
+		 * @param string $version       Version number of plugin (optional)
+		 * @param string $dependencies  a multidimensional array of dependencies  (optional)
 		 */
-		public function add_registered_plugin( $main_class, $version = null, $path = null, $dependencies = null ) {
+		public function add_registered_plugin( $main_class, $path, $version = null, $dependencies = array() ) {
```

Comment:
phpdoc parameters do not match, $version should be null|string and $dependencies should be array instead of string